sql/src/main/java/io/crate/planner/NoopPlan.java

```diff
@@ -38,6 +38,11 @@
     private NoopPlan(){
     }
 
+    @Override
+    public StatementType type() {
+        return StatementType.MANAGEMENT;
```

Comment:
I'm not 100% sure about this... maybe it should be ignored?
Currently statements like BEGIN & END fall here, but also some normal statements that semantically do a NoOp.

Reply:
CreateTable, DropTable, ... that use a NoOp should be classified as DDL
BEGIN/END would be UNCLASSIFIED I guess.
not sure what the best classification is

Reply:
I tend to choose UNCLASSIFIED for everything here rather than MANAGEMENT.
And imho, I don't see any value in trying to get the original statement that led to NoopPlan.